proposals/extension_get_contexts.md

```diff
@@ -0,0 +1,171 @@
+# New API: extension.getContexts()
+
+## Background / Summary
+
+Chromium currently has the
+<code>[extension.getViews()](https://developer.chrome.com/docs/extensions/reference/extension/#method-getViews)</code>
+API method which allows an extension to get information about the views the
+extension is running in. Common reasons for calling this method are: determining
+if a toolbar popup is open, reading and/or modifying a settings page, and etc.
```

Comment:
"Etc." is an abbreviation of "et cetera". This phrase comes from Latin where et means "and" and cetera means "the rest". As such, and should not appear before etc. or the expanded meaning would be "and and the rest".

```suggestion
if a toolbar popup is open, reading and/or modifying a settings page, etc.
```